### PR TITLE
@alloy => Exclude Year in Art slugs

### PR DESCRIPTION
**Pull request by kanaabe**
This makes use of NOT to exclude certain articles from being redirected to the iOS app.

### CODE REVIEW

File: apple-app-site-association.json [hunk 1]
```diff
@@ -18,7 +18,12 @@
     "details": [
       {
         "appID": "23KMWZ572J.net.artsy.artsy",
-        "paths": [ "*" ]
+        "paths": [ "NOT /article/artsy-editorial-the-most-influential-curators-of-2016",
+                   "NOT /article/article/artsy-editorial-the-top-emerging-artists-of-2016",
```
Comment (alloy) [inline apple-app-site-association.json]: What'a with the double '/article' pefix?

File: apple-app-site-association.json [hunk 2]
```diff
@@ -18,7 +18,12 @@
...
+                   "NOT /article/artsy-editorial-17-emerging-artists-to-watch-in-2017",
+                   "NOT /article/artsy-editorial-the most-influential-living-artists-of-2016",
```
Comment (alloy) [inline apple-app-site-association.json]: Should that white space be a hyphen?

**Review by alloy — CHANGES_REQUESTED** [review]

File: apple-app-site-association.json (hunk 1 above)
Comment (kanaabe) [reply apple-app-site-association.json]: Sorry both my bad -- 🙈

**Comment by alloy** [comment]
No worries! That's what reviews are for 👍

**Comment by artsyit** [comment]
:rocket: PR was released in [`v1.1.0`](https://github.com/artsy/artsy-eigen-web-association/releases/tag/v1.1.0) :rocket:

**Pull request merged**